db/schema.rb

```diff
@@ -142,6 +142,21 @@
     t.boolean  "active",        null: false
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "strategy_id"
+  end
+
+  create_table "self_care_reminders", force: :cascade do |t|
```

Comment:
Why are there two tables with the same name `self_care_reminders` and `selfcare_reminders` and are these tables even being used?